package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celonis/content-cli",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Can you pull latest `master` branch, because I think we are already at `0.3.2` version.